libs/base/ui/overlays/modal/src/component.ts

```diff
@@ -3,7 +3,7 @@ import { componentDef } from '@spryker-oryx/core';
 export const modalComponent = componentDef({
   name: 'oryx-modal',
   impl: () =>
-    window.HTMLDialogElement
```

Comment:
the same as above

Reply:
Using `globalThis` for now as discussed because of unit tests. Solution may appear later in another ticket.